### ISSUE TITLE
gui won't load on linux 

### ISSUE THREAD

**Issue by speedi3** [keybase/client#5598]
after a fresh install on devuan , I can log in from command line , I can browse /keybase , but not subfolders . GUI is stuck on "loading" .
I don't have the GUI issue on other distros , so it is probably devuan related .

my log id: 311cd56b04e1ba3b9840e81c

**Comment by cjb** [comment]
Yeah, it's somethng like a missing $XDG_RUNTIME_DIR env var. You're probably on your own with niche distros.
